api.go

```diff
@@ -234,6 +234,16 @@ type (
 		Status TaskStatus `json:"status"`
 	}
 
+	importAssetRequest struct {
```

Comment:
should this type also be exported? `ImportAssetRequest`? since the project is aimed to be a central library.